www/app/page.tsx

```diff
@@ -207,6 +230,11 @@ export default function Home() {
 
     const reader = data.body?.pipeThrough(new TextDecoderStream()).getReader()!;
 
+    const messageContainer = messageContainerRef.current;
```

Comment:
So from what I can tell about the logic here you are first scrolling too the bottom and then if the user scrolls up during rendering it won't continue to scroll. I'm wondering if that should be the behavior or just default to always checking

Reply:
i think this is standard:

- if you're at the bottom, you stay at the bottom,
- if you're reading something above, you stay above
- you can break out of the bottom lock at all times, by scrolling up